.vscode/settings.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
I am not sure that this file should be here since it is only an empty VSCode Config file 👀